runtime/onert/core/include/util/ShapeInference.h

```diff
@@ -156,6 +163,12 @@ class DynamicInferer : public ir::OperationVisitor
   void visit(const ir::operation::Tanh &op);
   // TODO write op starting from U
   // TODO write op starting from Z
+private:
+  /**
+   * @brief Performs shape inference and memory allocation for arithmetic operation
+   */
+  void handleArithmeticOp(const ir::Operation &op, const ir::OperandIndex lhs_idx,
```

Comment:
Does this have something to do with add, mul, sub, div operation?
If so, it would be good to refer to the name of BinaryArithmeticOp for matching code style.
This is only my opinion.

Reply:
Good